var/spack/repos/builtin/packages/everytrace/package.py

```diff
@@ -0,0 +1,25 @@
+# Install with:
+#	   spack diy --skip-patch everytrace@devel +fortran +mpi ^openmpi
+
+from spack import *
+import llnl.util.tty as tty
+import os
+
+class Everytrace(CMakePackage):
+	"""Get stack trace EVERY time a program exits."""
+
+	homepage = "https://github.com/citibeth/everytrace"
+	url		 = "https://github.com/citibeth/everytrace/tarball/dev"
+
+	version('devel', '0123456789abcdef0123456789abcdef')
```

Comment:
The URL I put in there (line 12) should fetch; GitHub can give you a tarball directly, if you tell it which branch/tag you want to fetch.  Of course, fetching the "dev" branch would be bad practice for anyone other than the developer.  Since the "spack diy" usage I outlined above doesn't try to fetch anyway, it would be more useful to put a real version tag in that URL in case someone is running Spack in regular mode.  But... if a project is early in development, there won't be such a tag yet; so might as well just fetch master or dev.

The checksum won't match yet, but that can be fixed easily enough.